low-level/files/opt/seagate/sspl/setup/sspl_setup_init.py

```diff
@@ -0,0 +1,189 @@
+#!/bin/env python3
+
+# Copyright (c) 2020 Seagate Technology LLC and/or its Affiliates
+#
+# This program is free software: you can redistribute it and/or modify it under the
+# terms of the GNU Affero General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE. See the GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License along
+# with this program. If not, see <https://www.gnu.org/licenses/>. For any questions
+# about this software or licensing, please email opensource@seagate.com or
+# cortx-questions@seagate.com.
+
+#################################################################
+# This script performs following operations.
+# - Creates datapath as defined in /etc/sspl.conf
+# - Check dependencies for consts.roles other than '<product>'
+#################################################################
+
+import sys
+import os
+import subprocess
+import pwd
+
+sys.path.insert(0, '/opt/seagate/cortx/sspl/low-level/')
+
+from framework.base.sspl_constants import file_store_config_path, roles
+from files.opt.seagate.sspl.setup.sspl_setup import Cmd as SSPLSetup
+import psutil
+
+import rpm 
```

Comment:
No it comes with python site_packages, need not to install any pip pkg.

Reply:
what python version you are using

Reply:
python 3.6.8 I am using but python2.7 onwards have rpm.

Reply:
It requires yum pkg,   `python36-rpm`  for python 3.6

Reply:
@ujjwalpl  need your approval here for use of new python pkg python36-rpm

Reply:
Can we use in-built packages ? like "import rpm" ? 
Ref: https://docs.fedoraproject.org/en-US/Fedora_Draft_Documentation/0.1/html/RPM_Guide/ch16s03s02.html

Reply:
One my machine, rpm exists and its not part of any package. Can you recheck

Reply:
@Sumedh-a-k confirmed with @ujjwalpl, we need to avoid using any new 3rd party pkgs, so we will use subprocess (or SimpleProcess from cortx utils ) to invoke direct rpm commands and parse the o/p